test/cache_invalidation_test.rb

```diff
@@ -46,14 +46,14 @@ def test_reload_invalidate_cached_objects
   def test_after_a_reload_the_cache_perform_as_expected
     Item.cache_has_many :associated_records, :embed => :ids
 
-    assert_equal [@baz, @bar], @record.associated_records
     assert_equal [@baz, @bar], @record.fetch_associated_records
+    assert_equal [@baz, @bar], @record.associated_records
```

Comment:
Does performing this assert `assert_equal [@baz, @bar], @record.associated_records` change anything in this case? Or was it to be consistent with the ordering of the asserts that are performed after the reload?